applications/ConstitutiveLawsApplication/python_scripts/symbolic_generation/hyper_elastic_simo_taylor_neo_hookean/generate_hyper_elastic_simo_taylor_neo_hookean.py

```diff
@@ -0,0 +1,65 @@
+import sympy
+import KratosMultiphysics.sympy_fe_utilities as KratosSympy
```

Comment:
This whole file do you think is necessary to have it in the repo? Just curiosity

Reply:
Sure it is. This is what we've always done with all the automatic differentiations. Besides, I think it would be nice to use it as it is but with different strain energies to generate the most common hyper-elastic models in literature.